src/lib/rest/RestService.cpp

```diff
@@ -475,7 +486,22 @@ std::string restService(ConnectionInfo* ciP, RestService* serviceV)
     commonFilters(ciP, &parseData, &serviceV[ix]);
     scopeFilter(ciP, &parseData, &serviceV[ix]);
 
+
     std::string response = serviceV[ix].treat(ciP, components, compV, &parseData);
+
+    //
+    // If we have gotten this far, and the status is 200, the Input is OK.
```

Comment:
Yes, that is the idea.
I couldn't find a better place to reset the badInput ...

By the way, a few leftover comments here - to be removed of course